### DISCUSSION TITLE
SSL - I stuck at "Generating SSL Certificate" for hours

### DISCUSSION THREAD

**Discussion by hhtoan** [vercel/community#3028] [Help] [answered]
### Summary

I have configured everything needed to integrate my domain with Vercel. However, I encountered a problem during the last step of generating SSL certificates, and I am unsure how to troubleshoot it.

After doing some research, I found some discussions suggesting that I should disable DNSSEC. Therefore, I would appreciate it if you could investigate whether DNSSEC is enabled on my domain, which is hhtoan.id.vn.

Best regards,

![image](https://github.com/vercel/community/assets/39783978/1548b73c-beae-4ede-bb8e-2bc246320c5f)

### Steps to Reproduce

None

**Comment by amyegan** [comment]
Hi @h2toan. Thanks for reporting this. We'll take a look and share more info as it becomes available 🙇 

~~Edit: We have an incident related to DNS and certificates - https://www.vercel-status.com/incidents/hyxqrbmy5mgj~~

**Reply by smaeda-ks** [reply, marked as answer]
@h2toan This is unrelated to the incident above.

See https://community.letsencrypt.org/t/some-vn-cctld-sld-arent-recognized-and-causing-ratelimited-error/200596

Let's Encrypt needs to update their policy
